src/utils/dbc.json

```diff
@@ -146,4 +146,4 @@
   "TOYOTA ALPHARD 2020": "toyota_nodsu_pt_generated",
   "TOYOTA ALPHARD HYBRID 2021": "toyota_nodsu_pt_generated",
   "VOLKSWAGEN PASSAT NMS": "vw_golf_mk4"
-}
+}
```

Comment:
originally the newline was here from me creating the file, but when the script generates the file there is newline